recipes-bsp/secure-boot-otp-provisioning/files/keys/custSmpk.crt

```diff
@@ -0,0 +1,35 @@
+-----BEGIN CERTIFICATE-----
+MIIGETCCA/mgAwIBAgIUbIdEpsn7FuWg5xLPuyQIq9UDiaYwDQYJKoZIhvcNAQEL
+BQAwgZcxCzAJBgNVBAYTAkNOMRAwDgYDVQQIDAdTaWNodWFuMRAwDgYDVQQHDAdD
+aGVuZ2R1MRMwEQYDVQQKDApTaWVtZW5zIEFHMQ0wCwYDVQQLDARTRVdDMRMwEQYD
+VQQDDApTaWVtZW5zIEFHMSswKQYJKoZIhvcNAQkBFhxJT1QyMDAwLmluZHVzdHJ5
+QHNpZW1lbnMuY29tMB4XDTIwMDIwODEzNDI1MVoXDTIwMDMwOTEzNDI1MVowgZcx
+CzAJBgNVBAYTAkNOMRAwDgYDVQQIDAdTaWNodWFuMRAwDgYDVQQHDAdDaGVuZ2R1
+MRMwEQYDVQQKDApTaWVtZW5zIEFHMQ0wCwYDVQQLDARTRVdDMRMwEQYDVQQDDApT
+aWVtZW5zIEFHMSswKQYJKoZIhvcNAQkBFhxJT1QyMDAwLmluZHVzdHJ5QHNpZW1l
+bnMuY29tMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAzHGdNypxRKEN
+Nyy23fbykrA2Jz7jHorj0KSZv/s2jCczpFIExXyShW5f4j54tDwZSF2sw1ukd9TM
+/pb3bLjxIUZWI7aMdOBUvYoiXTJdSRHOxOu+4lEoYyyhKrZHKVaPE/OKLD7sWHwL
+ovnVCVBMyG0ZfcaEnBKdAqxQ6rUGkg/uqMnz8KdKS2FrX+a/nR8WOSJewZxuhHkV
+XRMCMrEOjkSdXiWBn6+KQnzsDavrZ0BJwNStSZaxIY0UHK2cVNt1SOaHhSNyftKP
+NCTGQYEQOcg5y2oPC+w5EZQmjsZOhreBFt9w/36eY8w1UYwW/KQQhlpi8iIkRSvy
+STfvsdJJkzqfMGcgKTPYdOkyue5EamdTBGp4uF89jB+Jf4a5lVwUTlkxW16A73/d
+o9AekEf0tued+dPSIYhnuK1nNhNg25Ou8QL3MJQuIpEFaoX9BasxPfMq6XIHfi4V
+Yw9o07UqOhLc1/rMQHBBbaEWSINWxRo2XM7qrdpM3gJbGle1jFgGCf0jDFrMwRj4
+OPUWJlJY+LOgohb94QiKegztgJbRpZU6Cbyvr2qDbJUs0nFsOlGSLX3RpPqnEqX4
+JwXJGBqKhsSfrkf07Gp1FtvzNtz5beq59h2YQticST6NoCvc020gpQSGjAGqp0cF
+wcmNKfxaoK9tT5ctsPzwytp4mfIKBekCAwEAAaNTMFEwHQYDVR0OBBYEFCmdG69J
+wSWagtazdlFxxAB0YncYMB8GA1UdIwQYMBaAFCmdG69JwSWagtazdlFxxAB0YncY
+MA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggIBAHHPSOTgLeNLpcgj
++RA69yDBmp/+wHxF6P/HbEUcXyJPTrZIvWwiUOT16M5XT6Yr5FagnOwRE/p6rGxh
+YqcEk0spFqyAnFEAK+F4Ij0bWSodSn7TstYqUNu9hNehwclKWTNyR1a9Bo2uZwuj
+EAsvgZcS8aKJbRnTVco6peZ/y/e+56byP2qINq8G1OCCElpGMpSMnK8e3kIEFbXC
+EN0yPPidk5/6xqm4BcHgbo78PTetdP5DaytXczF9bxMG1L4yiKClLFDu1dexUk9K
+7ntP1e/Hpn7ZNoPaVdEW9dSdb6TlzzQpwWrV5JH2ItO1nK0Kv3ZPiwx5CmdIz4yt
+5ZtaHPGs0lpELT9cY6JvT0QApotW0+naOwO6pbtnFwD+f3YtFtFjX5HSdLW3yJBO
+pqJo80m6RyV2lthrzWwXI+MZbJScuRA9ouUq10INsydbAqLaTmSzBfBxoktammYv
+h47Q7SK+KEa4XNZiysr78rdhiYMZkrxH3UU12xeLGXGIVieVn1CwNU5fTMxAcssa
+v2fBFJbLi9Y8WYJE6Ce/bCBhPBDMOiu3vc7i5DWZoIkvcqFi0Xb0cOTGnxwyXAqN
+psPURMglyWtDAD5HAt5J6yfVCvqRGg0cbr8uND8ZI+Q2JPW1sJy8UrYTxah4VQX0
+06sNtd4LExh1/UWWyKmRM9PgOD0u
+-----END CERTIFICATE-----
```

Comment:
```
        Validity
            Not Before: Feb  8 13:42:51 2020 GMT
            Not After : Mar  9 13:42:51 2020 GMT
```
Please refresh first. Probably also the other cert (custMpk does not need that if you reuse the in-tree version).